docs/basics/tx-lifecycle.md

```diff
@@ -6,7 +6,7 @@
 
 ## Synopsis
 
-This document describes the lifecycle of a transaction from creation to committed state changes. Transaction definition is described in a [different doc](./transactions.md). The transaction will be referred to as `Tx`.
+This document describes the lifecycle of a transaction from creation to committed state changes. Transaction definition is described in a [different doc](../core/tx-msgs.md#transactions). The transaction will be referred to as `Tx`.
```

Comment:
do we want to include empty .md files? https://github.com/cosmos/cosmos-sdk/blob/master/docs/core/tx-msgs.md